Core/DailyPixelFiring.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  DailyPixelFiring.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public protocol DailyPixelFiring {
+    static func fireDaily(_ pixel: Pixel.Event,
+                          withAdditionalParameters params: [String: String])
+    
+    static func fireDaily(_ pixel: Pixel.Event)
+}
+
+extension DailyPixel: DailyPixelFiring {
+    static public func fireDaily(_ pixel: Pixel.Event) {
+        fire(pixel: pixel)
+    }
+
+    public static func fireDaily(_ pixel: Pixel.Event, withAdditionalParameters params: [String: String]) {
```

Comment:
ordering of the modifiers is not consistent

Reply:
https://github.com/duckduckgo/iOS/pull/3210/commits/df780128b3c732fd61dfd5116b6649ee6848ad16